src/d3a/models/strategy/external_strategy.py

```diff
@@ -113,9 +123,17 @@ def _serialize_offer_dict(offer_dict):
 
     def _offer_lists(self, payload):
         try:
-            return_data = self._serialize_offer_dict(self.market.offers)
-            self.publish(self._list_offers_response_channel,
-                         {"status": "ready", "offer_list": return_data})
+            if ConstSettings.IAASettings.MARKET_TYPE == 1:
+                filtered_offers = [{"id": v.id, "price": v.price, "energy": v.energy}
+                                   for _, v in self.market.offers.items()]
```

Comment:
So for the one sided market the user can see ALL offers, not only just the one she posted?

Reply:
Yes, because on one-sided markets a user will need to select from available offers in the market and accept it via the 'accept_offer' endpoint. If the user was not able to see other people's offers, he would not know which offer to accept.

Reply:
You are right of course! Thanks for explaining again!